.github/CHANGELOG.md

```diff
@@ -18,15 +18,17 @@
 ### Documentation
 
 ### Bug fixes
-  
+* Patch `MultiControlledX` tests to fix stable/latest failures after the v0.40.0 release.
```

Comment:
@AmintorDusko Could you also add the shots reduction to the changelog for the future reference?